src/clients/api/queries/getTransactions/formatTransaction.ts

```diff
@@ -1,31 +1,41 @@
 import BigNumber from 'bignumber.js';
-import { TransactionCategory, TransactionEvent } from 'types';
-import { convertTokensToWei, getTokenByAddress, getVTokenByAddress } from 'utilities';
+import { Token, TransactionCategory, TransactionEvent } from 'types';
+import { getTokenByAddress, getVTokenByAddress } from 'utilities';
 
 import { TOKENS } from 'constants/tokens';
 
 import { TransactionResponse } from './types';
 
 const formatTransaction = ({
-  amount,
+  amountMantissa,
   category,
   event,
   tokenAddress,
   timestamp,
+  from,
   ...rest
 }: TransactionResponse) => {
-  const token =
-    getTokenByAddress(tokenAddress) ||
-    getVTokenByAddress(tokenAddress)?.underlyingToken ||
-    TOKENS.xvs;
+  // check if the tokenAddress is from a VToken
+  const vToken = getVTokenByAddress(tokenAddress);
+
+  // if it is, use the VToken decimals and the image from the underlying token
+  const transactionToken: Token | undefined = vToken
+    ? { ...vToken.underlyingToken, decimals: vToken.decimals }
+    : // else get the token from tokenAddress
+      getTokenByAddress(tokenAddress);
+
+  // if neither is found, use XVS
+  const token = transactionToken || TOKENS.xvs;
 
   return {
     ...rest,
-    amountWei: convertTokensToWei({ value: new BigNumber(amount), token }),
+    amountMantissa: new BigNumber(amountMantissa),
```

Comment:
Why the name change from wei to mantissa?

Reply:
Particularly, it feels slightly better to use the same term from the payload. Otherwise it looks like we are dealing with two different amount values. Either that or we just use wei/mantissa interchangeably

Reply:
I see, that makes sense. We can start using mantissa on the frontend too: I think for now we'll consider wei/mantissa to be interchangeable as you mentioned and progressively move towards using mantissa instead of wei.